scene/animation/animation_mixer.cpp

```diff
@@ -629,12 +629,13 @@ bool AnimationMixer::_update_caches() {
 			if (!track) {
 				Ref<Resource> resource;
 				Vector<StringName> leftover_path;
-				Node *child = parent->get_node_and_resource(path, resource, leftover_path);
+				Node *child = parent->get_node_or_null(path);
 
 				if (!child) {
-					ERR_PRINT("AnimationMixer: '" + String(E) + "', couldn't resolve track:  '" + String(path) + "'.");
+					WARN_PRINT("AnimationMixer: '" + String(E) + "', couldn't resolve track:  '" + String(path) + "'.");
```

Comment:
It should be left as an error.

Reply:
Reverted back to error, changed to has_node_and_resource. Don't not want to retrieve the object and then check, as it would result in an extra error being posted when retrieving the object with get_node_and_resource.